nodes/nodes/nodes-sno-worker-nodes.adoc

```diff
@@ -0,0 +1,69 @@
+:_content-type: ASSEMBLY
+[id="nodes-sno-worker-nodes"]
+= Worker nodes for {sno} clusters
+:context: add-workers
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+{sno-caps} clusters reduce the host prerequisites for deployment to a single host. This is useful for deployments in constrained environments or at the network edge. However, sometimes you need to add additional capacity to your cluster, for example, in telecommunications and network edge scenarios. In these scenarios, you can add worker nodes to the single-node cluster.
+
+Adding worker nodes does not expand the cluster control plane, and it does not provide high availability to your cluster. For {sno} clusters, high availability is handled by failing over to another site.
+
+It is not recommended to add a large number of worker nodes to a single-node cluster.
+
+[NOTE]
+====
+Unlike multi-node clusters, by default all ingress traffic is routed to the single control-plane node, even after adding additional worker nodes.
+====
+
+== Adding worker nodes to {sno} clusters
+
+There are several ways that you can add worker nodes to a single-node cluster. You can add worker nodes on link:https://console.redhat.com/openshift/assisted-installer/clusters[console.redhat.com], you can add worker nodes to clusters manually, or you can use the Assisted Installer REST API directly.
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about worker node hardware requirements, see xref:../../installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc#installation-minimum-resource-requirements_installing-restricted-networks-bare-metal[Minimum resource requirements for cluster installation].
```

Comment:
I would suggest for all the xrefs , to use a similar format as per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections.  For example

- xref:../../installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc#installation-minimum-resource-requirements_installing-restricted-networks-bare-metal[Minimum resource requirements for cluster installation].